guides/release/object-model/computed-properties.md

```diff
@@ -84,66 +85,73 @@ However, you may find that you have to observe the properties of a different obj
 For example, look at this computed property:
 
 ```javascript
-import EmberObject, { computed } from '@ember/object';
+import EmberObject, { computed } from "@ember/object";
 
-let home = EmberObject.extend({
-  location: {
-    streetName: 'Evergreen Terrace',
+class Home extends EmberObject {
+  location = {
+    streetName: "Evergreen Terrace",
     streetNumber: 742
-  },
+  }
 
-  address: computed('location.streetName', 'location.streetNumber', function() {
+  @computed("location.streetName", "location.streetNumber")
+  address() {
     return `${this.location.streetNumber} ${this.location.streetName}`;
-  })
-});
+  }
+}
+
+let home = Home.create();
 
-home.address // 742 Evergreen Terrace
-home.set('location.streetNumber', 744)
-home.address // 744 Evergreen Terrace
+home.address; // 742 Evergreen Terrace
+home.set("location.streetNumber", 744);
+home.address; // 744 Evergreen Terrace
 ```
 
 It is important to observe an object's properties, not the object itself that has properties nested inside. If the object reference `location` is used as a dependent key, the computed property will not recalculate when the `streetName` or `streetNumber` properties change.
 
 ```javascript
-import EmberObject, { computed } from '@ember/object';
+import EmberObject, { computed } from "@ember/object";
 
-let home = EmberObject.extend({
-  location: {
-    streetName: 'Evergreen Terrace',
+class Home extends EmberObject {
+  location = {
+    streetName: "Evergreen Terrace",
     streetNumber: 742
-  },
+  }
 
-  address: computed('location', function() {
+  @computed("location")
+  get address() {
     return `${this.location.streetNumber} ${this.location.streetName}`;
-  })
-});
+  }
+}
+
+let home = Home.create();
```

Comment:
this important instantiation step is missing from the current _release_ guides (not octane preview). I'll follow on with a PR to fix this in master